### PR TITLE
T-63 Code style conventions

### PR DESCRIPTION
**Pull request by Dimirolz**

### CODE REVIEW

**Review by IgVol — CHANGES_REQUESTED** [review]
please change commit and branch naming convention

Commits:
T-ticket_number Description, where description starts with capital letter verb.
f.e:
T-69 Add comments to pull request

Branches:
we aren't following gitflow in any way, so we can skip using feature/ prefixes in branches, and instead write like
t-69-add-comments-to-pull-request
We use dash without uppercase.

Pull requests should be named same as task title in Trello starting with task number.
F.e.
T-69 Review pull requests

Please note, we always use Present Simple. F.e use "Add" instead of "added". We do not use # symbol when referencing tickets, because GitHub references it like link to PR.

please make sure everybody starts following naming conventions.

Please make sure we using either .js or .jsx files exclusively.

We store images in /public folder with possible exception of svg react files.

File: README.md [hunk 1]
```diff
@@ -50,9 +50,17 @@ You should check your open pull-requests at least once a day and in case if any
...
+You need to install a pre-commit hook, to execute `eslint` checks automatically on every commit. <br/>
 Run `npm run prepare` to install husky pre-commit hook properly.
 
+### Style conventions
+
+1. We are using `PropTypes` for all our components that have props.
+2. All components folders must be named in CamelCase. 
+3. All components must be exported with index.js file inside component folder. 
+4. We are using upperCase for class naming in our CSS modules.
+5. We do not shrink the names of imports. Examples: "styles", not "s". "classNames", not "cn", etc...
```
[the remark this answers is not in the document]
Comment (Dimirolz) [reply README.md:62]: @IgVol 
Questions: 
1. Do we really need to name commits with task numbers? They are inside branch with task number, already.
2. "Please make sure we usу either .js or .jsx files exclusively." - we need somehow change this rule. We have api routes, utils. They are always be .js

[the remark this answers is not in the document]
Comment (IgVol) [reply README.md:62]: 1. Yes. 
2. I wan't clear enough: I meant using .js or .jsx for react components. I'd always use .js

[the remark this answers is not in the document]
Comment (Dimirolz) [reply README.md:62]: @IgVol 
1. I cannot understand the reason. Can you explain it to me?
2. Why not .jsx? I guess for files with jsx usage is more convinient using .jsx filetype

**Review by IgVol — APPROVED** [review]

**Review by ZakharovMaxim — APPROVED** [review]

**Pull request merged**